requirements.txt

```diff
@@ -0,0 +1,12 @@
+colcon-ros==0.3.14
```

Comment:
Can you show me where in the ROS2 docs it says that requirements.txt is supported or recommended in ROS2 as an alternative to rosdep?

Reply:
Well, officially `rosdep` shall be used to manage requirements and dependencies, which are described in `package.xml`. However, I personally hate don't like when some tooling changes my base system (rosdep uses apt-get to install dependencies). I prefer to have my source isolated. That's why I provide this requirements.txt file, whilch allows me to run rosbridge within a venv. 

However, if you prefer, I can delete the file from this repo. 

Please let me know your preference.

Reply:
The main reason I don't want requirements.txt in the repo is because it becomes everybody else's job to maintain your requirements.txt.  We already have one list of dependencies to maintain.

Reply:
I have deleted the requirements.txt file from this fork.